package.json

```diff
@@ -46,5 +46,8 @@
     "ignore": [
       "nunjucks"
     ]
+  },
+  "devDependencies": {
+    "cookie-parser": "^1.4.3"
```

Comment:
should this be dependencies, not devDependencies? Is this a different approach to the one taken in the beta repo?

Reply:
I think the dependencies for the kits is a bit of a mess, I'll move this into dependencies for now but they're definitely not all regular dependencies...